README.md

```diff
@@ -1075,3 +1075,217 @@ Response:
 ```
 <FILE>
 ```
+
+
+## HOSTS
+#### GET Host data
+GET /disaster-recovery/v1beta1/virtual-continuous-host-protectors/
+
+Response:
+```
+{
+    "items":
+    [
+        {
+            "id": "d7cf09d2-16db-35e3-b552-e43dcfe4d974",
+            "customerId": "d293d71c-0b4b-11ee-ab1f-9a4e6a06796a",
+            "address": "soluta",
+            "clusterId": "",
+            "version": "",
+            "associatedSite": {
+                "id": "a54a5a61-2cc6-42cc-8468-8094d0595326",
+                "name": "ut",
+                "address": "185.229.31.138",
+                "resourceUri": "/disaster-recovery/v1beta1/virtual-sites/a54a5a61-2cc6-42cc-8468-8094d0595326"
+            },
+            "associatedVra": {
+                "name": "Z-VRA-soluta",
+                "ipAddress": "86.132.122.254",
+                "version": "8.0",
```

Comment:
Change the version to have 3 octets, with double digits on the last one, like 8.0.10

Reply:
fixed